ush/global_cycle.sh

```diff
@@ -234,6 +234,7 @@ HOMEgfs=${HOMEgfs:-$BASEDIR/gfs_ver.${gfs_ver}}
 EXECgfs=${EXECgfs:-$HOMEgfs/exec}
 FIXfv3=${FIXfv3:-$HOMEgfs/fix/fix_fv3_gmted2010/$CASE}
 FIXam=${FIXam:-$HOMEgfs/fix/fix_am}
+FIXsst=${FIXsst:-/scratch4/NCEPDEV/da/noscrub/Xu.Li/data/rtgsst/fix}
```

Comment:
New FIXsst files need to be added to $HOMEgfs/fix/fix_am directory on all platforms

Reply:
FIXsst  has been changed to be FIXam (are you looking at the newest version?)